grammar.js

```diff
@@ -80,10 +80,16 @@ module.exports = grammar({
       '{}',
       /\{\{+[0-9]*/,
       '(',
+      ')',
+      '[',
+      ']',
+      "['",
+      "']",
```

Comment:
Dropping `')'` and `']'` avoids 10 (out of ~30) new errors in a full `gen_help_html.validate()` run. And still passes the [new tests](https://github.com/neovim/tree-sitter-vimdoc/pull/103/files#diff-334e21593203e1c6431a705d373c69ad292ce95f8a3cc78fa032239819e01aec).

```suggestion
      '[',
      "['",
      "']",
```